src/style.css

```diff
@@ -1,3 +1,68 @@
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+  padding: 0 50px;
+  /* min-width: 1024px; */
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__link {
+  display: inline-block;
+  text-decoration: none;
+  color: black;
+  margin-right: 20px;
+  height: 60px;
+  line-height: 60px;
+  text-transform: uppercase;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+}
+
+.nav :hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to the CSS variable.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)